privaterelay/middleware.py

```diff
@@ -8,11 +10,61 @@
 from django.shortcuts import redirect
 
 import markus
+from csp.middleware import CSPMiddleware
 from whitenoise.middleware import WhiteNoiseMiddleware
 
 metrics = markus.get_metrics("fx-private-relay")
 
 
+# To find all the URL paths that serve HTML which need the CSP nonce:
+# python manage.py collectstatic
+# find staticfiles -type f -name 'index.html'
+CSP_NONCE_COOKIE_PATHS = [
+    "/",
+    "/contains-tracker-warning",
+    "/flags",
+    "/faq",
+    "/vpn-relay/waitlist",
+    "/accounts/settings",
+    "/accounts/profile",
+    "/accounts/account_inactive",
+    "/vpn-relay-welcome",
+    "/phone/waitlist",
+    "/phone",
+    "/404",
+    "/tracker-report",
+    "/premium/waitlist",
+    "/premium",
```

Comment:
The trailing slash is needed: 
```suggestion
    "/contains-tracker-warning/",
    "/flags/",
    "/faq/",
    "/vpn-relay/waitlist/",
    "/accounts/settings/",
    "/accounts/profile/",
    "/accounts/account_inactive/",
    "/vpn-relay-welcome/",
    "/phone/waitlist/",
    "/phone/",
    "/404/",
    "/tracker-report/",
    "/premium/waitlist/",
    "/premium/",
```